### PR TITLE
Add tooling for conversion to/from Markdown

### PR DESCRIPTION
**Pull request by Zash**
## xep → markdown

```bash
xmllint --nonet --noent --loaddtd xep-0313.xml | lua > xep-0313.md
```

Requires [luaexpat](https://matthewwild.co.uk/projects/luaexpat/).

## markdown  → xep

```bash
pandoc xep-0313.md -t tools/2xep.lua
```

Requires [pandoc](https://pandoc.org/).

### CODE REVIEW

File: tools/2xep.lua [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+-- are piped through graphviz and images are included in the HTML
+-- output using 'data:' URLs.
+--
+-- Invoke with: pandoc -t sample.lua
+--
+-- Note:  you need not have lua installed on your system to use this
+-- custom writer.  However, if you do have lua installed, you can
+-- use it to test changes to the script.  'lua sample.lua' will
+-- produce informative error messages if your code contains
+-- syntax errors.
```
Comment (horazont) [inline tools/2xep.lua:13]: @Zash Wanna fix or remove this heading?

**Review by horazont — COMMENTED** [review]
:partying_face: This is awesome! Can you check on that comment?

File: tools/2xep.lua (hunk 1 above)
Comment (Zash) [reply tools/2xep.lua]: Oh glob licenses, how do they work?

**Comment by Zash** [comment]
**poke**

**Pull request merged**